src/api/HydroCodeLookup.cs

```diff
@@ -0,0 +1,72 @@
+﻿using BDMS.Models;
+
+namespace BDMS;
+
+/// <summary>
+/// The HydroCodeLookup static class provides mappings between <see cref="Hydrotest.TestKindId"/> identifiers
+/// and compatible geolcodes for hydrotest results, flow directions, and evaluation methods.
+/// </summary>
+/// <remarks>
+/// The mappings are used to ensure the compatibility of test results, flow directions,
+/// and evaluation methods with specific test kinds.
+/// </remarks>
+public static class HydroCodeLookup
+{
+    /// <summary>
+    /// Gets a dictionary that maps a <see cref="Hydrotest.TestKindId"/>   identifier to a list of compatible geolcodes for the <see cref="HydrotestResult.Parameter"/> of a <see cref="HydrotestResult"/>.
```

Comment:
PP: Unnötigen Whitespace löschen
```suggestion
    /// Gets a dictionary that maps a <see cref="Hydrotest.TestKindId"/> identifier to a list of compatible geolcodes for the <see cref="HydrotestResult.Parameter"/> of a <see cref="HydrotestResult"/>.
```